open-company/open-company-web
# Doo slimer tests

card: https://trello.com/c/JYtz8oQG

to test:
- `lein test!`
- check that [travis](https://travis-ci.org/open-company/open-company-web) is testing w/o errors
